### PR TITLE
MGMT-8722: Populate EventsURL in InfraEnv

### PR DESCRIPTION
**Pull request by slaviered**
# Assisted Pull Request

## Description

Populate EventsURL in InfraEnv

## List all the issues related to this PR

- [x] New Feature
- [ ] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [ ] Cloud
- [x] Operator Managed Deployments
- [ ] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [x] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [ ] No tests needed

## Assignees

/cc @
/cc @

## Checklist

- [ ] Title and description added to both, commit and PR.
- [ ] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [ ] Reviewers have been listed
- [ ] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- [ ] Are the title and description (in both PR and commit) meaningful and clear?
- [ ] Is there a bug required (and linked) for this change?
- [ ] Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: internal/controller/controllers/common.go [hunk 1]
```diff
@@ -258,3 +262,25 @@ func machineNetworksEntriesToArray(entries []hiveext.MachineNetworkEntry) []*mod
 		return &models.MachineNetwork{Cidr: models.Subnet(entry.CIDR)}
 	}).([]*models.MachineNetwork)
 }
+
+func eventsURL(baseURL string, authType auth.AuthType, signParams gencrypto.CryptoPair, filterBy ...string) (string, error) {
```
Comment (nmagnezi) [inline internal/controller/controllers/common.go:266]:
```suggestion
func generateEventsURL(baseURL string, authType auth.AuthType, signParams gencrypto.CryptoPair, filterBy ...string) (string, error) {
```

**Review by nmagnezi — CHANGES_REQUESTED** [review]
just one comment for now, until the PR is ready.

**Comment by rollandf** [comment]
looks good

**Pull request merged**